README.md

```diff
@@ -829,6 +829,24 @@ actions.Exec.executeCommandLine(time.Duration.ofSeconds(20), 'echo', 'Hello Worl
 
 See [JS-Joda](https://js-joda.github.io/js-joda/) for more examples and complete API usage.
 
+### Parsing and Formatting
+
+Occasionally, one will need to parse a non-supported date time string or generate one from a ZonedDateTime.
+To do this you will use [JS-Joda DateTimeFormatter and potentially your Locale](https://js-joda.github.io/js-joda/manual/formatting.html).
+However, all the locales combine to a unacceptibly large size to ship them with the openhab-js library.
```

Comment:
```suggestion
However, shipping all the locales with the openhab-js library would lead to an unacceptable large size of it.
```

Reply:
I accepted the rewording but cut off the "of it" part. I don't think it's needed.